src/transformers/onnx/config.py

```diff
@@ -75,19 +75,20 @@ class OnnxConfig(ABC):
     torch_onnx_minimum_version = version.parse("1.8")
     _tasks_to_common_outputs = {
         "default": OrderedDict({"last_hidden_state": {0: "batch", 1: "sequence"}}),
```

Comment:
I decided to reorder these features alphabetically to make it easier to find the one you need during development